test/unit/preact/test-base-element-mapping.js

```diff
@@ -132,6 +135,7 @@ describes.realWin('PreactBaseElement', {amp: true}, (env) => {
       element.setAttribute('enabled', '');
       element.removeAttribute('disabled');
       element.setAttribute('part-b', 'C');
+      element.setAttribute('data-param-test', 'worldhello');
```

Comment:
Hmm. I don't understand how this test would pass since I don't see the implementation. I also made a comment about the same in the `base-element.js`.